hack/verify-external-dependencies-version.sh

```diff
@@ -26,6 +26,16 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 kube::golang::verify_go_version
 
-cd "${KUBE_ROOT}"
+# Ensure that we find the binaries we build before anything else.
+export GOBIN="${KUBE_OUTPUT_BINPATH}"
+PATH="${GOBIN}:${PATH}"
 
-go run cmd/verifydependencies/verifydependencies.go "${KUBE_ROOT}"/build/dependencies.yaml
+# Install levee
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
```

Comment:
You could also use the POSIX-compatble way (more portable, less writing, simpler) of achieving the same thing you need here:
```shell
cd "${KUBE_ROOT}/hack/tools"
GO111MODULE=on go install sigs.k8s.io/zeitgeist
cd -
```

Reply:
@joakimr-axis -- Thanks for the suggestion! Pulled it in as part of https://github.com/kubernetes/kubernetes/pull/98845/commits/71008247dcf816062596efe35f9cb36edcf43164.